src/Storage/Adapters/WindowsAzure.Storage.9/AzureContextAdapterExtensions.cs

```diff
@@ -0,0 +1,135 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
These changes shouyld not go in the common repo - they shouls dimply go in the storage module in the azure-powershell repo